### DISCUSSION TITLE
Why my github student developer pack is not being verified ? its about to be a month still not verified

### DISCUSSION THREAD

**Discussion by nabinyd** [community/community#86991] [GitHub Education]
### student developer not being verified,

i have used campus gmail for student developer pack and its not being verified for student developer pack and been almost a month. its only showing "we are still processing" but getting nothing. please help to verify my account

**Comment by soummyaanon** [comment]
okay I can guide You  Can you please apply again and try to send the proper obvious image of your college ID card your secondary email should be your college email try this hopefully your problem going to be solved Thank you.
